apps/ui/src/components/FormSpaceController.vue

```diff
@@ -0,0 +1,56 @@
+<script lang="ts" setup>
+import { shorten } from '@/helpers/utils';
+import { Network } from '@/networks/types';
+
+defineProps<{
+  disabled: boolean;
+  controller: string;
+  network: Network;
+}>();
+
+const emit = defineEmits<{ (e: 'save', value: string) }>();
```

Comment:
```suggestion
const emit = defineEmits<{
  (e: 'save', value: string);
}>();
```

nit.

Reply:
Is there an eslint rule to enforce that ?